addons/website_slides/views/slide_channel_add.xml

```diff
@@ -5,18 +5,19 @@
     <field name="name">slide.channel.view.form.add</field>
     <field name="model">slide.channel</field>
     <field name="arch" type="xml">
-        <form js_class="website_slides_add_form">
+        <form js_class="website_new_content_form">
             <group>
+                <div class="oe_title">
+                    <label for="name" string="Course Title"/>
+                    <h1><field name="name" placeholder="e.g. Computer Science for kids"/></h1>
+                </div>
```

Comment:
I don't think it make sense to have an oe_title inside a group, is not it ouside everywhere else?

Reply:
I thought it was normal since already used in code ([e.g](https://github.com/odoo/odoo/blob/f4a8d5bf9b77b04aa86b492941177b3fc147aa8d/addons/account/views/account_full_reconcile_views.xml#L10-L11))... The idea here was to give `oe_title` a full width since `.o_group` > `.o_field_widget` has this, but I can change it and add a `w-100` on the field... what do you think?

Reply:
If the oe_title class has a limited width, that's because that's the style we want, no? Is there a problem with it not being full width? It's like 70% or something like that no?

Reply:
I just did a quick check on the code and this seems more complicated than what I thought, actually it's related to the new Owl views structure:
For example, in the case of `oe_title`, on legacy views, we had the field as `input` in the DOM which makes it full width (since `[type="text"]{ width: 100% }` in CSS). Now on Owl views, the field is a `<div...><input/></div>` which explains the input is no longer full width
I don't know if it was intended when views were changed to Owl but the goal here is just to have enough width to display the input placeholder:
![screenshot-localhost_8069-2022 09 02-13_10_28](https://user-images.githubusercontent.com/25032876/188127329-ecd72d24-8701-4924-9226-da7708069b77.png)